rpc/wrtc_server.go

```diff
@@ -43,6 +44,51 @@ type webrtcServer struct {
 
 	onPeerAdded   func(pc *webrtc.PeerConnection)
 	onPeerRemoved func(pc *webrtc.PeerConnection)
+
+	counters struct {
```

Comment:
I think this an awesome start for WebRTC stats + will not recommend any others for the time being.